cmake/jakt-executable.cmake

```diff
@@ -8,8 +8,12 @@ set(JAKT_TARGET_IN "${CMAKE_SYSTEM_PROCESSOR}-unknown-${CMAKE_SYSTEM_NAME}-unkno
 STRING(TOLOWER ${JAKT_TARGET_IN} JAKT_TARGET)
 
 function(add_jakt_compiler_flags target)
+  if (MSVC)
+    target_compile_options("${target}" PRIVATE -W[123])
```

Comment:
this should just be /W4.

Extra space below in the -Wall branch.

instead of doing this, can you add /W4 and -Wall to the branches on lines 43 and 45 below with the other MSVC check?